docs/guides/ha/cluster-setup.md

```
@@ -0,0 +1,388 @@
+Within a few seconds of starting, you should see one node win the election:
+
+```
+INF raft: entering candidate state  node=node-1
+INF raft: election won               tally=3
+INF raft: entering leader state      leader=node-1
+INF block produced                   height=1 hash=0xabc...
+```
+
+The other nodes will log:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)